pkg/config/fetch.go

```diff
@@ -7,7 +7,8 @@ type FetchOptions struct {
 	// SkipDeps is the skip deps flag
 	SkipDeps bool
 	// OutputDir is the output directory
-	OutputDir string
+	OutputDir         string
+	OutputDirTemplate string
```

Comment:
and comments.

Reply:
What kind of comment? `// OutputDirTemplate is the output directory template` or `// OutputDirTemplate is the ouptut dir flag` – similar to the other FetchOptions?

Reply:
Something like this?
```
// OutputDirTemplate is the go template to generate the path of output directory
```

Reply:
Thank you for the suggestion!